Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -363,6 +367,11 @@ function initialize(model) {
   model._texturesLoadedPromise = texturesLoadedPromise
     .then(function () {
       model._texturesLoaded = true;
+
+      // Re-run the pipeline so texture memory statistics are re-computed
+      if (loader._incrementallyLoadTextures) {
+        model.resetDrawCommands();
```

Comment:
We talked about this offline and decided this test would be not worth the effort. This one is difficult to test because A. it happens in the middle of private promise chains and B. it's hard to distinguish it from other things that reset the draw commands.